Tasks/SonarQubePostTest/PRCA/PostComments-Module.psm1

```diff
@@ -24,6 +24,8 @@
 
 #region Public Constants
 
+. $PSScriptRoot/PostComments-Server.ps1
```

Comment:
I moved the functions that interact with the server to a separate file as the module now has 600+ lines.